.github/workflows/deploy-ghpages-latest-stable.yml

```diff
@@ -57,40 +57,14 @@ jobs:
   # TODO: this job should have a own reusable workflow
   build:
     
-    runs-on: ubuntu-latest
     needs: check-label
-    steps:
+    uses: ./.github/workflows/build-docs.yml
+    with:
+      python-version: ${{ inputs.python-version }}
+      package-manager: ${{ inputs.package-manager }}
+      dependency-path: ${{ inputs.dependency-path }}
```

Comment:
You forgot to pass down `path-doc`: you might get one from the user, so you want to pass it also to the further workflow.